### ISSUE TITLE
Source Visibility Variable Missing

### ISSUE THREAD

**Issue by ukiews** [bitfocus/companion-module-obs-studio#288]
**Describe the bug**
While there is an option to toggle source visibility, there is no variable exposed to monitor specific source visibility.

**To Reproduce**
See Variables > obs

**Version Information:**

- Operating System: MacOS
- OBS Version: 30.2.2
- Companion Version: 3.4.0+7290

**Comment by bryce-seifert** [comment]
There is not a variable, but there is a feedback “ Source Enabled in Scene”, does that work for your needs?

**Comment by ukiews** [comment]
> There is not a variable, but there is a feedback “ Source Enabled in Scene”, does that work for your needs?

No, since I'm trying to use it with the generic blink module which requires a variable.

**Comment by bryce-seifert** [comment]
Hi, there is now a built-in blink function in Companion, so you can do something like this with the Source Enabled in Scene feedback to achieve this result

<img width="571" height="510" alt="Image" src="https://github.com/user-attachments/assets/e1b9a2b6-582e-465e-b6d7-5f4b82e25e83" />
